src/brevitas/graph/equalize.py

```diff
@@ -107,15 +116,46 @@ def _get_input_axis(module):
 
 
 def _get_output_axis(module):
-    if isinstance(module, (torch.nn.Linear, torch.nn.Conv1d, torch.nn.Conv2d, torch.nn.Conv3d)):
+    if isinstance(module, (torch.nn.Linear, torch.nn.Conv1d, torch.nn.Conv2d, torch.nn.Conv3d, torch.nn.BatchNorm1d, torch.nn.BatchNorm2d, torch.nn.BatchNorm3d)):
         return 0
     elif isinstance(module, (torch.nn.ConvTranspose1d, torch.nn.ConvTranspose2d, torch.nn.ConvTranpose3d)):
         return 1
     else:
         raise RuntimeError(f"Module {module} not supported.")
 
 
-def _cross_layer_equalization(srcs, sinks):
+def _combine_weights_bias(m, axis, bias_shrinkage):
+    if m.bias is None:
+        return m.weight.data
+
+    bias = m.bias.data
+    weight = m.weight.data if axis == 0 else m.weight.data.transpose(1,0)
+    weight = deepcopy(weight).view(weight.shape[0], -1)
+    bias = deepcopy(bias).view(-1, 1)
+
+    weight = torch.where(torch.abs(weight) < 1e-8, torch.tensor(1e-8), weight)
+    factor = torch.abs(bias)/torch.abs(weight)
+
+    # From https://github.com/Xilinx/Vitis-AI/blob/master/src/vai_quantizer/vai_q_pytorch/nndct_shared/optimization/commander.py#L450
+    if bias_shrinkage == 'vaiq':
+        if torch.abs(bias).max() < 10 and (torch.abs(bias).max()/torch.abs(weight).max() ) < 20:
+            if torch.median(factor) > 100 or torch.mean(factor) > 1000:
+                shrink_factor = 5
+            else:
+                shrink_factor = 2
+        else:
+            if  torch.median(factor) > 30 or torch.mean(factor) > 500:
+                shrink_factor = 20
+            elif torch.median(factor) > 15 or torch.mean(factor) > 250:
+                shrink_factor = 10
+            else:
+                shrink_factor = 5
+    else:
```

Comment:
```
elif isinstance(bias_shrinkage, (int, float)):
    ....
else:
   raise RuntimeError(f"{bias_shrinkage} not supported.")
```

Always check the correctness of each case, and raise an error otherwise.